fork_choice/confirmation-rule.md

```diff
@@ -0,0 +1,639 @@
+
+This section specifies an algorithm to determine whether a block is confirmed.
+
+### Constants
+
+The following values are (non-configurable) constants used throughout this specification.
+
+| Name                                            | Value     | Description                                                                                                                                                                                                                                                                                                          |
+|-------------------------------------------------|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `UNCONFIRMED_SCORE`                             | `int(-1)` | Value returned by the `get_*_score` methods to indicate that the block passed in input cannot be confirmed even if all validators are honest.                                                                                                                                                                        |
+| `MAX_CONFIRMATION_SCORE`                        | `int(33)` | Maximum possible value of the confirmation score corresponding to the maximum percentage of Byzantine stake.                                                                                                                                                                                                         |
+| `COMMITTEE_WEIGHT_ESTIMATION_ADJUSTMENT_FACTOR` | `int(5)`  | Per mille value to add to the estimation of the committee weight across a range of slots not covering a full epoch in order to ensure the safety of the confirmation rule with high probability. See [here](https://gist.github.com/saltiniroberto/9ee53d29c33878d79417abb2b4468c20) for an explanation about the value chosen. |
+
+### Configuration
+
+The confirmation rule can be configured to the desired tolerance of Byzantine validators, for which the algorithm takes the following input parameters:
+
+| Input Parameter                    | Type     | Max. Value               | Description                                                                                             |
+|------------------------------------|----------|:-------------------------|---------------------------------------------------------------------------------------------------------|
+| `CONFIRMATION_BYZANTINE_THRESHOLD` | `uint64` | `MAX_CONFIRMATION_SCORE` | assumed maximum percentage of Byzantine validators among the validator set.                             |
+| `CONFIRMATION_SLASHING_THRESHOLD`  | `Gwei`   | `2**64 - 1`              | assumed maximum amount of stake that the adversary is willing to get slashed in order to reorg a block. |
```

Comment:
I am a bit confused by using different units for two similar parameters. I can appreciate the argument that an amount of stake adversary is willing to slash is a good security indicator in this case, but:
1) Paper operates with ratios in both cases
2) Amount in gwei can be easily obtained from total active balance and percentage
3) The confirmation score is in percentage, it is valuable to use total active balance to compute an amount of stake that adversary would need to reorg a block

I am curious what others thinking on it?